GVRf/Framework/framework/src/main/java/org/gearvrf/GVRSticker.java

```diff
@@ -0,0 +1,150 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf;
+
+import android.graphics.Bitmap;
+import android.os.Environment;
+import android.util.Log;
+
+import org.gearvrf.utility.Threads;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+/**
+ * This class takes screenshots at a specified interval and writes the output PNG files to GearVRFScreenshots directory.
+ * The capturing can be stopped by calling {@link GVRSticker#stop()}
+ * The images can be used to create Stickers (a GIF animation)
+ */
+
+public class GVRSticker {
+    private final GVRContext mGVRContext;
+    private final String mTag;
+    private final String mDirectory;
+    private final long mInterval;
+    private boolean mCaptureFlag = true;
+    private final static String TAG = "GVRSticker";
+
+    /**
+     * Constructor takes the following parameters
+     * @param gvrContext
+     * @param tag
+     *           Name for outputting PNG's to the SD Card along with appended frame ID
+     * @param interval
+     *           Time in milliseconds to take screenshot at that interval
+     */
+    public GVRSticker(GVRContext gvrContext, String tag, long interval){
+        mGVRContext = gvrContext;
+        mTag = tag;
+        mInterval = interval;
+
+        File sdcard = Environment.getExternalStorageDirectory();
+        mDirectory = sdcard.getAbsolutePath() + "/GearVRFScreenshots/";
+        File d = new File(mDirectory);
+        d.mkdirs();
```

Comment:
Probably the fact that WRITE_EXTERNAL_STORAGE is required should be mentioned. Also I'd recommend checking the return value of mkdirs and throwing an exception immediately if needed.